language.cto

```diff
@@ -0,0 +1,483 @@
+  o Creoles
+  o Kashubian
+  o CushiticLanguages
+  o Czech
+  o Dakota
+  o Danish
+  o Dargwa
+  o LandDayaklanguages
+  o Delaware
+  o Dogrib
+  o Dinka
+  o Divehi
+  o Dogri
+  o DravidianLanguages
+  o Duala
+  o Dutch
+  o Dyula
+  o Dzongkha
+  o Efik
+  o Ekajuk
+  o Elamite
+  o English
+  o Esperanto
+  o Estonian
+  o Ewe
+  o Ewondo
+  o Fang
+  o Faroese
+  o Fanti
+  o Fijian
+  o Filipino
+  o Finnish
+  o Finno-UgrianLanguages
+  o Fon
+  o French
+  o Northern Frisian
+  o Eastern Frisian
+  o Western Frisian
+  o Fulah
+  o Friulian
+  o Ga
+  o Gayo
+  o Gbaya
+  o GermanicLanguages
+  o Georgian
+  o German
+  o Geez
+  o Gilbertese
+  o Gaelic
+  o Galician
+  o Manx
+  o Gondi
+  o Gorontalo
+  o Gothic
+  o Grebo
+  o Guarani
+  o SwissGerman
+  o Gujarati
+  o Gwich'in
+  o Haida
+  o Haitian
+  o Hausa
+  o Hawaiian
+  o Hebrew
+  o Herero
+  o Hiligaynon
+  o HimachaliLanguages
+  o Hindi
+  o Hittite
+  o Hmong
+  o HiriMotu
+  o Croatian
+  o UpperSorbian
+  o Hungarian
+  o Hupa
+  o Iban
+  o Igbo
+  o Icelandic
+  o Ido
+  o SichuanYi
+  o IjoLanguages
+  o Inuktitut
+  o Interlingue
+  o Iloko
+  o Interlingua
+  o IndicLanguages
+  o Indonesian
+  o Indo-EuropeanLanguages
+  o Ingush
+  o Inupiaq
+  o IranianLanguages
+  o IroquoianLanguages
+  o Italian
+  o Javanese
+  o Lojban
+  o Japanese
+  o Judeo-Persian
+  o Judeo-Arabic
+  o Kara-Kalpak
+  o Kabyle
+  o Kachin
+  o Kalaallisut
+  o Kamba
+  o Kannada
+  o KarenLanguages
+  o Kashmiri
+  o Kanuri
+  o Kawi
+  o Kazakh
+  o Kabardian
+  o Khasi
+  o KhoisanLanguages
+  o CentralKhmer
+  o Khotanese
+  o Kikuyu
+  o Kinyarwanda
+  o Kirghiz
+  o Kimbundu
+  o Konkani
+  o Komi
+  o Kongo
+  o Korean
+  o Kosraean
+  o Kpelle
+  o Karachay-Balkar
+  o Karelian
+  o KruLanguages
+  o Kurukh
+  o Kuanyama
+  o Kumyk
+  o Kurdish
+  o Kutenai
+  o Ladino
+  o Lahnda
+  o Lamba
+  o Lao
+  o Latin
+  o Latvian
+  o Lezghian
+  o Limburgan
+  o Lingala
+  o Lithuanian
+  o Mongo
+  o Lozi
+  o Luxembourgish
+  o Luba-Lulua
+  o Luba-Katanga
+  o Ganda
+  o Luiseno
+  o Lunda
+  o Luo
+  o Lushai
+  o Macedonian
+  o Madurese
+  o Magahi
+  o Marshallese
+  o Maithili
+  o Makasar
+  o Malayalam
+  o Mandingo
+  o Maori
+  o AustronesianLanguages
+  o Marathi
+  o Masai
+  o Malay
+  o Moksha
+  o Mandar
+  o Mende
+  o Irish
+  o Mi'kmaq
+  o Minangkabau
+  o UncodedLanguages
+  o Mon-KhmerLanguages
+  o Malagasy
+  o Maltese
+  o Manchu
+  o Manipuri
+  o ManoboLanguages
+  o Mohawk
+  o Mongolian
+  o Mossi
+  o MultipleLanguages
+  o MundaLanguages
+  o Creek
+  o Mirandese
+  o Marwari
+  o MayanLanguages
+  o Erzya
+  o NahuatlLanguages
+  o NorthAmericanIndianLanguages
+  o Neapolitan
+  o Nauru
+  o Navajo
+  o NdebeleSouth
+  o NdebeleNorth
+  o Ndonga
+  o Low German
+  o Nepali
+  o NepalBhasa
+  o Nias
+  o Niger-KordofanianLanguages
+  o Niuean
+  o NorwegianNynorsk
+  o BokmålNorwegian
+  o Nogai
+  o Norse
+  o Norwegian
+  o N'Ko
```

Comment:
I'm surprised that `-` and `'` are valid here. Did you edit this is the VSCode extension for that it is checked?